### PR TITLE
Added missing jwt-decode@1 types. Added generic for jwt-docode.

### PR DESCRIPTION
**Pull request by GiedriusGrabauskas**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/17049
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by madsmadsen — APPROVED** [review]

File: types/jwt-decode/jwt-decode-tests.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
- 
-import jwtDecode = require('jwt-decode');
- 
-let token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJmb28iOiJiYXIiLCJleHAiOjEzOTMyODY4OTMsImlhdCI6MTM5MzI2ODg5M30.4-iaDojEVl0pJQMjrbM1EzUIfAZgsbK_kgnVyVxFSVo";
+import * as jwtDecode from "jwt-decode";
```
Comment (minestarks) [inline types/jwt-decode/jwt-decode-tests.ts]: The old import statement was correct as ```jwt-decode``` isn't an ES6 module ([explanation](https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this))
```ts 
import jwtDecode = require('jwt-decode')
```

**Review by minestarks — CHANGES_REQUESTED** [review]

File: types/jwt-decode/jwt-decode-tests.ts (hunk 1 above)
Comment (GiedriusGrabauskas) [inline types/jwt-decode/jwt-decode-tests.ts]: It's the same. Read the section "I'm Sad About This; Rules are Dumb." about "hack" in your explanation.

Comment (DovydasNavickas) [inline types/jwt-decode/jwt-decode-tests.ts]: I think @GiedriusGrabauskas is right. The imports are the same.

**Review by DovydasNavickas — APPROVED** [review]

**Comment by GiedriusGrabauskas** [comment]
@minestarks ?

**Pull request merged**